libcrux-ml-kem/pqcp/Cargo.toml

```diff
@@ -0,0 +1,48 @@
+[package]
+name = "libcrux-ml-kem"
+version = "0.0.2-pre.2"
+authors = ["Cryspen"]
+license = "Apache-2.0"
+homepage = "https://github.com/cryspen/libcrux"
+edition = "2021"
+repository = "https://github.com/cryspen/libcrux"
+readme = "Readme.md"
+exclude = ["/tests"]
```

Comment:
What about the other excludes?

Reply:
The other excludes I don't include in `pqcp/allowlist.txt`, so they are not copied over to the PQCA repository. I was wondering about the value of this field, though, in the context of the PQ code package, since we will not re-publish from there anyway, right?

Reply:
Good point. I was thinking about keeping things in sync. It shouldn't matter indeed.

Reply:
The new, generated `Cargo.toml` does not include this field.